activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   `Mysql2Adapter#quote_string` now raises `ActiveRecord::ConnectionNotEstablished` rather than
+    `ActiveRecord::InvalidStatement` when it can't connect to the MySQL server.
```

Comment:
Can we mention the change here also affects to all adapters as I concerned at https://github.com/rails/rails/pull/40158#issuecomment-685571996?

> #40152 is breaking not only `@connection.quote` in mysql2 adapter but also `@connection.execute` in all adapters, I've concerned about the latter part.

Reply:
Sure I'll add this, but as a second entry, otherwise it might be a bit confusing.

Reply:
Updated. Let me know if it's acceptable now.